src/main/main.ts

```diff
@@ -551,9 +567,10 @@ function initializeIpc() {
     utils.deleteBlockchainStoreSync(getBlockChainStorePath());
     if (rerun) {
       if (useRemoteHeadless) {
-        initializeRemoteHeadless();
+        console.log("main clear cache call initializeRemoteHeadless");
+        await initializeRemoteHeadless();
       } else {
-        initializeHeadless();
+        await initializeHeadless();
```

Comment:
Just a curiosity, what does this do?

Reply:
It seems previous work remain. I'll delete it.